app/views/admin/services/_history.html.erb

```diff
@@ -4,7 +4,7 @@
       </div>
       <div class="row">
     <% end %>
-    <div class="col-md-4 data-set">
+    <div class="col-md-4 data-set" id="<%="version-#{set.version}"%>">
```

Comment:
I use it here:
https://github.com/alphagov/imminence/blob/add-soft-validation-for-snac-codes/features/step_definitions/service_steps.rb#L113